src/apprt/gtk/notebook.zig

```diff
@@ -27,6 +27,8 @@ pub const Notebook = union(enum) {
 
         // Create a notebook to hold our tabs.
         const notebook_widget: *c.GtkWidget = c.gtk_notebook_new();
+        c.gtk_widget_add_css_class(notebook_widget, "notebook");
```

Comment:
I think a good refactor here would be to add this call in the `create()` function.